### PR TITLE
Quote(s) -> pluralized properly

### PR DESCRIPTION
**Pull request by MrOnosa**
It's a pet-peeve of mine.

### CODE REVIEW

**Comment by MrOnosa** [comment]
That's what I get for trying to program on my cell phone.

**Comment by Ixrec** [comment]
For what it's worth, if we want to be fancy about plural handling, what we'd really want is a little helper method like `pluralize(number, "thing", "things")`. It may be a trivial method, but over all the "(s)"s in a codebase like this (or in the one at my day job which does have such a helper) having a standard name for it and slightly less logic at the string interpolation site adds up to a significant benefit.

**Comment by MrOnosa** [comment]
And that's what I get for not just forking the project because it was working locally but I forgot to update all the files using the GitHub editor.

Noted on using pluralize. Were you thinking of a full library, like [Pluralize.NET](https://github.com/sarathkcm/Pluralize.NET), or just a simple helper method?

**Comment by Ixrec** [comment]
Oh I just meant a simple helper method for us. I don't think Izzy's usage of plural words is nearly advanced or dynamic enough to benefit from what Pluralize.NET is doing.

File: Izzy-MoonbotTests/Tests/QuoteModuleTests.cs [hunk 1]
```diff
@@ -72,6 +72,31 @@ public async Task QuoteCommand_Tests()
         // TODO: test cases where user left (both cached in userinfo and not)
     }
 
+    [TestMethod()]
+    public async Task QuotesCommand_Tests()
```
Comment (Ixrec) [inline Izzy-MoonbotTests/Tests/QuoteModuleTests.cs]: Test name should mention the whole plural thing it's trying to test

File: Izzy-MoonbotTests/Tests/QuoteModuleTests.cs [hunk 2]
```diff
@@ -72,6 +72,31 @@ public async Task QuoteCommand_Tests()
...
+    {
+        // setup
+
+        var (cfg, _, (_, sunny), _, (generalChannel, _, _), guild, client) = TestUtils.DefaultStubs();
+        DiscordHelper.DefaultGuildId = guild.Id;
+
+        // only one quote so that the "random" selection is deterministic for now
```
Comment (Ixrec) [inline Izzy-MoonbotTests/Tests/QuoteModuleTests.cs]: this comment is incorrect in this test

**Review by Ixrec — CHANGES_REQUESTED** [review]
just a few more small fixes

**Review by Ixrec — APPROVED** [review]

**Comment by ParaSwarm** [comment]
@MrOnosa you're a peach and a warrior of light

**Pull request merged**